src/Magnum/OvrIntegration/Session.cpp

```diff
@@ -37,6 +37,29 @@
 
 namespace Magnum { namespace OvrIntegration {
 
+namespace Implementation {
+/**
+@brief HMD status flag
+
+@see @ref HmdStatusFlags
+*/
+enum class HmdStatusFlag: UnsignedByte {
+    /**
+     * A mirror texture was created for the HMD and needs to be destroyed on
+     * destruction of the HMD
+     */
+    HasMirrorTexture = 1 << 0,
+
+    /** The HMD was created as a debug HMD (without real hardware) */
+    Debug = ovrHmdCap_DebugDevice /* 0x0010 */
+};
```

Comment:
:+1: for the idea of having the enum definition only in the `*.cpp` file. A new thing for me! :)